Sources/FluentUI_iOS/Components/Navigation/NavigationBar.swift

```diff
@@ -354,7 +354,14 @@ open class NavigationBar: UINavigationBar, TokenizedControl, TwoLineTitleViewDel
     private var titleStyleObserver: NSKeyValueObservation?
 
     private let backButtonItem: UIBarButtonItem = {
-        let backButtonItem = UIBarButtonItem(image: UIImage.staticImageNamed("back-24x24"),
+        let backButtonImage: UIImage?
+        if #available(iOS 26, *) {
+            backButtonImage = UIImage(systemName: "chevron.backward")
```

Comment:
though testing I've noticed apple will do special things if you use `barButtonSystemItem` so can we actually use `UIBarButtonItem(barButtonSystemItem: .back ...)`. this should also handle accessibility for us but I'd double check

Reply:
jks I was thinking of close. there's no `.back`